fish/config.fish

```diff
@@ -26,7 +52,7 @@ set PATH /usr/local/sbin $PATH
 set PATH /usr/local/bin $PATH
 
 # Android
-set -x ANDROID_HOME /usr/local/share/android-sdk
+set -x ANDROID_HOME ~/Library/Android/sdk
```

Comment:
Hmm, I suppose we should go with android-studio's install path instead of homebrew. (Sigh! I'll miss you CLI, and.. Most likely, will regret going away from CLI tools)